assignments/prototypes.js

```diff
@@ -15,13 +15,25 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+function GameObject(attributes) {
+  this.createdAt = attributes.createdAt;
+  this.name = attributes.name;
+  this.dimensions = attributes.dimensions;
+};
+GameObject.prototype.destroy = () =>  {
+  return `${this.name} was removed from the game.`;
+};
 
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+function  CharacterStats(statAttributes) {
+  this.healthPoints = statAttributes.healthPoints;
+  GameObject.call(this.attributes)
+}
```

Comment:
So you want your `this` keyword binding needs to be the first thing you do, so move @35 above the rest of the attributes. You need to reference the parent `GameObject` first.

Your `call()` syntax is wrong. You called it on the right parent `GameObject`, but it should be taking 2 arguments here. the first being `this` and the second (separated by comma) being the parameters for the child constructor in this case `statAttributes`